taskchampion/src/server/local.rs

```diff
@@ -1,104 +1,141 @@
 use crate::server::{
     AddVersionResult, GetVersionResult, HistorySegment, Server, VersionId, NO_VERSION_ID,
 };
-use crate::utils::Key;
-use kv::msgpack::Msgpack;
-use kv::{Bucket, Config, Error, Integer, Serde, Store, ValueBuf};
+use anyhow::Context;
+use rusqlite::params;
+use rusqlite::types::{FromSql, ToSql};
+use rusqlite::OptionalExtension;
 use serde::{Deserialize, Serialize};
 use std::path::Path;
 use uuid::Uuid;
 
+// FIXME: Duplicated
```

Comment:
Duplicated with taskchampion/src/storage/sqlite.rs?  That should be relatively easy to fix..

Reply:
Ah true, I think at the time I (half-)thought "server.rs -> sync-server -> different crate" so it would have had to be `pub`.. but both are in the same crate :roll_eyes: Fixed!